FGEGraphics/GraphicsHelpers/Renderer.cs

```diff
@@ -155,6 +159,39 @@ void GenerateBoxVBO()
         Box = builder.Generate();
     }
 
+    /// <summary>Generates a circle.</summary>
+    void GenerateCircleVBO()
+    {
+        const int segments = 32;
+        const int vertexCount = segments + 2;
+        const int indexCount = segments + 2;
+        Renderable.ArrayBuilder builder = new();
+        builder.Prepare2D(vertexCount, indexCount);
+        builder.Tangents = new Vector3[vertexCount];
+        builder.Vertices[0] = new Vector3(0, 0, 0);
+        builder.TexCoords[0] = new Vector3(0.5f, 0.5f, 0);
+        builder.Normals[0] = new Vector3(0, 0, -1);
+        builder.Colors[0] = new Vector4(1, 1, 1, 1);
+        builder.Tangents[0] = new Vector3(1, 0, 0);
+        builder.Indices[0] = 0;
+        // Generate vertices around circle
+        for (int i = 0; i <= segments; i++)
```

Comment:
I believe only the indices should have 1 extra, not the vertices -- there's a center, and 32 vertices, only the index needs to repeat `1` at the end to close the loop